packages/std/extra/pkg_config_make_paths_relative.bri

```diff
@@ -2,6 +2,28 @@ import * as std from "/core";
 import { runBash } from "./run_bash.bri";
 
 export const PKG_CONFIG_MAKE_PATHS_RELATIVE_SCRIPT = std.indoc`
+  make_pkg_config_paths_relative () {
+    local -r file="$1"
```

Comment:
In bash, we can declare variables to be local to a function, and we can declare it as readonly with `local -r`